### PR TITLE
Correct guest user tests closes #108

### LINKED ISSUE #108: correct guest_user_tests

### PR DESCRIPTION
**Pull request by oorion**
* Updated guest user tests to reflect new functionality.
* Left image test commented out.  Will pair with Jeff on this right now.
* Moved admin related tests within guest_user_test.rb to platform_admin_test.rb

### CODE REVIEW

File: test/integration/platform_admin_test.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
```
Comment (oorion) [inline test/integration/platform_admin_test.rb]: This was generated using rails g integration_test.  How about them apples hound?

File: app/views/tenants/projects/show.html.erb [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 
-<%= image_tag '#' %>
+<%= image_tag "#" %>
 <h2><%= "RETIRED ITEM" if @project.retired %></h2>
 <div id="description"><%= @project.description %></div>
 <div id="price"><%= number_to_currency @project.price / 100 %></div>
 
 <% unless @project.retired %>
-  <%= form_for :cart, url: carts_path do |f| %>
+  <%= form_for :cart, url: pending_loan_path do |f| %>
```
Comment (Jwan622) [inline app/views/tenants/projects/show.html.erb]: good correction. We're using singular routes now for the pending_loan

File: test/factories/factories.rb [hunk 3]
```diff
@@ -43,11 +43,11 @@
   end
 
   factory :order do
-    total_cost 8900
+    sequence(:total_cost) { |n| 10000 + n * 100 }
```
Comment (Jwan622) [inline test/factories/factories.rb]: This is dope. Good way to correct new total_costs using factory girl.

File: test/integration/guest_user_test.rb [hunk 4]
```diff
@@ -5,94 +5,67 @@ class GuestUserTest < ActionDispatch::IntegrationTest
...
 
-    assert page.has_content?(tenant.projects.first.title)
-    refute page.has_content?(non_tenant_project.title)
+    visit root_path
+    click_link_or_button(project1.categories.first.name)
+    click_link_or_button(project1.title)
+
+    assert_equal tenant_project_path(
+      slug: project1.tenant.organization,
+      id: project1.id
```
Comment (Jwan622) [inline test/integration/guest_user_test.rb]: Great test. Looks good and makes sense to me.

**Pull request merged**